src/Service/CloudWatchLogs/src/Result/PutLogEventsResponse.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace AsyncAws\CloudWatchLogs\Result;
+
+use AsyncAws\CloudWatchLogs\ValueObject\RejectedLogEventsInfo;
+use AsyncAws\Core\Response;
+use AsyncAws\Core\Result;
+
+class PutLogEventsResponse extends Result
+{
+    /**
+     * The next sequence token.
+     */
+    private $nextSequenceToken;
+
+    /**
+     * The rejected events.
+     */
+    private $rejectedLogEventsInfo;
+
+    public function getnextSequenceToken(): ?string
```

Comment:
Not sure if the casing is as expected on those methods, but they are autogenerated
guess this can only be fixed in the generator?

Reply:
Indeed, AWS uses lowers case to define properties for this service (while all other services are UpperCamelCase).

I think this is something we should fix in generator. /cc @Nyholm

Reply:
Hm. Yeah. Maybe. But in a different PR.

Reply:
Fixed in master. 

You can rebase if you want to. The bot will fix it if we merge this PR without a rebase.